protocol/implementations/js/src/protocolDefinitions.ts

```diff
@@ -0,0 +1,121 @@
+export const aliceProtocolDefinition = {
```

Comment:
So we're going with two separate definitions as a mechanism of enforcing roles? Is that the idea?

Reply:
yep, that's the idea! 

for this use case of two parties having distinct roles, and as dwn protocol definition is currently designed, we _must_ create two separate protocol definitions.

this is because if we had 1 protocol for both Alice and PFI, it's possible that Alice can write a Quote to PFI, or PFI can write an RFQ to Alice.

i do not think this is a great approach long term. i would like to just write one protocol and have it shared among multiple parties (that may play multiple distinct roles). this would require some more thinking on how dwn protocol definition is built in future versions.